zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -379,6 +395,111 @@ components:
           items:
             type: string
           nullable: true
+    JobActivationRequest:
+      type: object
+      properties:
+        type:
+          description: >
+            the job type, as defined in the BPMN process (e.g. <zeebe:taskDefinition
+            type="payment-service" />)
+          type: string
+        worker:
+          description: the name of the worker activating the jobs, mostly used for logging purposes
+          type: string
+        timeout:
+          description: >
+            a job returned after this call will not be activated by another call until the
+            timeout (in ms) has been reached
+          type: integer
+          format: int64
+        maxJobsToActivate:
+          description: the maximum jobs to activate by this request
+          type: integer
+          format: int32
+        fetchVariable:
+          description: >
+            a list of variables to fetch as the job variables; if empty, all visible variables at
+            the time of activation for the scope of the job will be returned
+          type: array
+          items:
+            type: string
+        requestTimeout:
+          description: >
+            The request will be completed when at least one job is activated or after the
+            requestTimeout (in ms). If the requestTimeout = 0, a default timeout is used.
+            If the requestTimeout < 0, long polling is disabled and the request is completed
+            immediately, even when no job is activated.
+          type: integer
+          format: int64
+        tenantIds:
+          description: a list of IDs of tenants for which to activate jobs
+          type: array
+          items:
+            type: string
+    JobActivationResponse:
+      description: The list of activated jobs
+      type: object
+      properties:
+        jobs:
+          type: array
+          items:
+            $ref: "#/components/schemas/ActivatedJob"
+    ActivatedJob:
+      type: object
+      properties:
+        key:
+          description: the key, a unique identifier for the job
+          type: integer
+          format: int64
+        type:
+          description: the type of the job (should match what was requested)
+          type: string
+        processInstanceKey:
+          description: the job's process instance key
+          type: integer
+          format: int64
+        bpmnProcessId:
+          description: the bpmn process ID of the job process definition
+          type: string
+        processDefinitionVersion:
+          description: the version of the job process definition
```

:upside_down_face: Same as above
```suggestion
          description: the version of the job's process definition
```